content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -455,6 +456,72 @@ From Prisma 3.0.0 onwards, the `prisma introspect` command is deprecated and rep
 
 </Admonition>
 
+### <inlinecode>validate</inlinecode>
+
+Validates the Prisma schema file.
+
+#### Arguments
+
+| Argument   | Required | Description                                                                                                                                         | Default                                     |
+| ---------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
+| `--schema` | No       | Specifies the path to the desired `schema.prisma` file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`, `./prisma/schema.prisma` |
```

Comment:
Alternative version, making it active voice:

```suggestion
| `--schema` | No       | Specifies the path to a `schema.prisma` file that Prisma will validate instead of the file at the default path. You can specify an absolute or relative path. | `./schema.prisma`, `./prisma/schema.prisma` |
```

Reply:
Would be good to do that in a separate PR, because it's the same text all over the page (10 times)